pkg/controller/acicontainersoperator.go

```diff
@@ -684,3 +827,235 @@ func (c *Controller) handleDaemonsetDelete(obj interface{}) bool {
 	log.Infof("aci-containers Daemonset Deleted")
 	return false
 }
+
+// intialize the dnsoperator client,
+// computes the dnsSpec.
+// local cache for all the routes will be updated.
+//  if there is change in the dns Spec, triggers the update
+func (c *Controller) updatednsOperatorCr() error {
+	log.Info("Update dnsoperator cr")
+	dnsInfo := &operatorv1.DNS{
+		TypeMeta:   metav1.TypeMeta{APIVersion: operatorv1.GroupVersion.String(), Kind: "DNS"},
+		ObjectMeta: metav1.ObjectMeta{Name: "default"},
+	}
+	cfg, err := config.GetConfig()
+	scheme := runtime.NewScheme()
+	err = operatorv1.Install(scheme)
+	if err != nil {
```

Comment:
Here and a bunch of places below, any error logging required?

Reply:
i am logging it in one single place  in the caller of this function. problem is i am seeing duplicate messages